source/man/doveadm-mailbox.1.rst.in

```diff
@@ -109,6 +109,68 @@ already deleted.
 **-8**
    Lists the mailboxes with UTF-8 encoding.
 
+mailbox metadata list
+---------------------
+
+**mailbox metadata list** [**-A** | **-u** *user* | **-F** *file*] [**-S** *socket_path*] [**-s**] [**-p**] *mailbox* [*key-prefix*]
+
+**-s**
+   Allow empty mailbox name.
```

Comment:
this actually means to use mailbox root instead of mailbox. So not exactly same thing. here and elsewhere.